R/ui.R

```diff
@@ -19,8 +19,8 @@ NULL
 #'   conveniently constructed via the `style` argument and `...`. See
 #'   'Examples'.
 #' @param filetype Vector of file extensions indicating which file types should
-#'   be styled. Case is ignored, and the `.` is optional, e.g. `c(".R", ".Rmd")`
-#'   or `c("r", "rmd")`.
+#'   be styled. Case is ignored, and the `.` is optional, e.g. `c(".R", ".Rmd",
+#'   ".Rnw")` or `c("r", "rmd", "rnw")`.
```

Comment:
Can you make adjustments in the description of this function in line 8 too (we have not even adopted it to `.Rmd` yet)?

Reply:
Also here: https://github.com/r-lib/styler/pull/434/commits/6b56ba48d41f30fb66b6949abac73955c7263d2b#diff-4f632064d1ef846baad1bfd3b7fc8dffR149

Reply:
💯